tembo-operator/tests/integration_tests.rs

```diff
@@ -65,6 +65,112 @@ mod test {
     const TIMEOUT_SECONDS_POD_DELETED: u64 = 300;
     const TIMEOUT_SECONDS_COREDB_DELETED: u64 = 300;
 
+    /// Struct to contain many commonly used test resources
+    /// 
+    /// Most if not all tests use all of the fields here, or refer to them
+    /// in some manner. This struct helps combine everything together.
+    struct TestClass {
```

Comment:
I also like this setup, but I would just change the naming of this, perhaps to "TestFixture". Also I am being perhaps picky, but I would just remove all mentions of the word "class", as we don't literally have classes is Rust. The word "Fixture" I think is the generic word used for what this is doing.

Reply:
I've opted for `TestCore` just to get rid of the nomenclature confusion. I couldn't find any consensus on the word "Fixture" in the context of Rust, but perhaps my keywords were flawed if this is a more generic-concept.